lib/inferno/entities/test_group.rb

```diff
@@ -9,8 +10,18 @@ class TestGroup
       extend DSL::HTTPClient::ClassMethods
       extend DSL::Runnable
       include DSL::FHIRValidation
+      include DSL::Results
+      include DSL::Assertions
+      include DSL::Messages
 
-      def_delegators 'self.class', :title, :id, :groups, :inputs, :outputs, :tests
+      def_delegators 'self.class', :title, :id, :block, :groups, :inputs, :outputs, :tests
+
+      attr_accessor :result_message, :child_results
```

Comment:
You think it's worth calling this `child_results` rather than just `results`?

Reply:
I just wanted to add clarity that these are the children's results being used, but I'm totally fine simplifying it to `results` since it only handles those anyway.